demo/conjur-client-files/policies/root.yml

```diff
@@ -0,0 +1,31 @@
+- !variable
```

Comment:
I think this would be easier to read if it followed the flow of our other policies: define roles, then resources, then do `permit`s. Example: https://github.com/conjurinc/conjurops-policies/blob/master/policy/ci/jenkins.yml

Also, `annotations` could be used for the roles (`layer`, `host`, `group`) and resources (`variable`). Adding annotations makes policies self-documenting and easier to understand in the UI.